tools/lib/provision.py

```diff
@@ -142,6 +142,13 @@
     "libappindicator1",
     "xdg-utils",
     # Puppeteer dependencies end here.
+
+    # Fixes missing sasl.h headers during
+    # python-ldap installation.
+    "libsasl2-dev",
```

Comment:
Does this belong in `VENV_DEPENDENCIES`?

Reply:
Actually, I am not sure if we need it in `COMMON_YUM_VENV_DEPENDENCIES` or not.